cupy/cusolver.py

```diff
@@ -0,0 +1,192 @@
+import numpy
+
+import cupy
+from cupy.cuda import cusolver
+from cupy.cuda import runtime
+from cupy.cuda import device
+
+
+_available_cuda_version = {
+    'gesvdj': (9000, None),
+    'gesvda': (10010, None),
+}
+
+
+def check_availability(name):
+    if name not in _available_cuda_version:
+        msg = 'No available version information specified for {}'.name
+        raise ValueError(msg)
+    version_added, version_removed = _available_cuda_version[name]
+    cuda_version = runtime.runtimeGetVersion()
+    if version_added is not None and cuda_version < version_added:
+        return False
+    if version_removed is not None and cuda_version >= version_removed:
```

Comment:
Agree to have `version_removed` to align with `cupy/cusparse.py`.
https://github.com/cupy/cupy/blob/2d1054919cec36c6065880f9e9391bbea051781c/cupy/cusparse.py#L82-L93